django/cantusdb_project/main_app/views/chant.py

```diff
@@ -724,6 +730,14 @@ def get_queryset(self) -> QuerySet:
 
             keyword_filter = ms_spelling_filter | std_spelling_filter | incipit_filter
             queryset = queryset.filter(keyword_filter)
+        if notes := self.request.GET.get("indexing_notes"):
+            operation = self.request.GET.get("indexing_notes_op")
+            # the operation parameter can be "contains" or "starts_with"
+            if operation == "contains":
+                indexing_notes_filter = Q(indexing_notes__icontains=notes)
+            else:
```

Comment:
My only question here is about this `else`, and not doing a check that the operation = "starts_with"... I'm only thinking about this in relation to the kind of thing in #1484. Is this the kind of place you are envisioning checking? Or was your thought about #1484 to do that separately.

Reply:
Ok, great! I'm good to merge